antora-ui-camel/src/css/frontpage.css

```diff
@@ -217,6 +222,10 @@ section.frontpage h1 {
   width: 27rem;
 }
 
+.icons8 {
+  width: 100px !important;
```

Comment:
Is the use of `!important` here justified? Rules in this file need to be scoped to `.frontpage`. Not sure about `icons8` class name, can it be more descriptive of what it represents?

Reply:
@zregvart yes i used this because images found on the home page have a default width of 27rem which is too big for that part. So I needed to add !important

Reply:
About icons8 it's the name for this Icon types. Actually I used Icons8 types. Only png types from 1px to 100px are free. SVG are not free. Should I consider using another Icon type ?